pjmedia/src/pjmedia/vid_stream.c

```diff
@@ -2392,6 +2392,22 @@ PJ_DEF(pj_status_t) pjmedia_vid_stream_send_rtcp_bye(
 }
 
 
+/*
+ * Send RTCP PLI.
+ */
+PJ_DEF(pj_status_t) pjmedia_vid_stream_send_rtcp_pli(
+						pjmedia_vid_stream *stream)
+{
+    PJ_ASSERT_RETURN(stream, PJ_EINVAL);
+
+    if (stream->enc && stream->transport) {
```

Comment:
as it is supposed to be decoder that sends PLI, shouldn't it be 'stream->dec' instead of 'stream->enc'?

Reply:
TBH I copied it from the `pjmedia_vid_stream_send_rtcp_bye` function. Come to think of it, I think we only need to check for stream->transport, since none enc or dec are used in send_rtcp. What do you thing?

Reply:
Just checked send_rtcp(), it seems to use stream->enc for logging purpose, not really sure why, will try to find out later. Anyway, only checking for stream->transport sounds fine. Thanks.